superset-frontend/src/explore/components/ExploreChartPanel/index.jsx

```diff
@@ -31,6 +31,7 @@ import {
   t,
   useTheme,
 } from '@superset-ui/core';
+import { checkIsDEXContext } from '@pinterest-plugins/src/dex/utils';
```

Comment:
With the current setup, it seems it will have circular dependency? the open source repo will depend on the internal repo(@pinterest-plugins/src/dex/utils'), and internal repo will also depend on open source repo

I think it should be one way dependency. the open source project should be able to be built and run on its own.

And also, can we try to not call out Dex in the open source repo?   I mean make it general if possible.
like for the checking of `!isDEXContext` below to determine if it should show something or not, we may have a config like showDataTablesPane , showConnectedExploreChartHeader , etc
basically, the open source repo is not supposed to know any specific feature from the internal repo